.github/workflows/publish_vscode_plugin.yml

```diff
@@ -0,0 +1,75 @@
+name: Publish VS Code Plugin
```

Comment:
I'd love to make this actions workflow simply build the plugin artifact and make it available on every push to master.

That will make it easy to publish manually with everything else, and make it easier to review changes to `harper-ls` and the plugin when pull requests are made.

I'd change the naming from "publish" to "package" and remove the actual publish step. See my other comment.